src/FixedPointDecimals.jl

```diff
@@ -38,6 +38,31 @@ import Base: reinterpret, zero, one, abs, sign, ==, <, <=, +, -, /, *, div, rem,
 
 include("fldmod_by_const.jl")
 
+# =========================================================
+# Set Int128 to widen to Int256 within FixedPointDecimals to avoid allocations from BigInt.
+# ---------------------------------------
+
+import BitIntegers
+
+# BitIntegers is missing unsigned(::Type{Int256})
+Base.unsigned(x::T) where T<:BitIntegers.XBS = reinterpret(unsigned(T), x)
+Base.unsigned(::Type{T}) where T<:BitIntegers.XBS = typeof(convert(Unsigned, zero(T)))
+
+# BitIntegers is missing iseven/isodd
+# Prevent expensive calculation for Int256
+Base.isodd(a::BitIntegers.Int256) = Base.isodd(a % Int)  # only depends on the final bit! :)
```

Comment:
Is this used?  If not consider removing?

Reply:
Ah yes it is. I've added a comment about why it's used. (`_round_to_even` calls `iseven`, which in turn calls `isodd`.)